phat/contracts/brick_profile/lib.rs

```diff
@@ -438,11 +468,21 @@ mod brick_profile {
                 Ok(account)
             }
         }
+
+        fn ensure_dumped_external_account(&self, id: ExternalAccountId) -> Result<ExternalAccount> {
+            let account = self.ensure_external_account(id)?;
+            if account.enabled || account.account_type != ExternalAccountType::Dumped {
```

Comment:
Why to check `enabled` here?

Reply:
Just double check, in case in the future we introduce another account type. Can remove.